engine/classes/Elgg/Notifications/NotificationsService.php

```diff
@@ -55,31 +44,19 @@ class NotificationsService {
 	/**
 	 * Constructor
 	 *
-	 * @param SubscriptionsService $subscriptions Subscription service
-	 * @param Queue                $queue         Queue
-	 * @param PluginHooksService   $hooks         Plugin hook service
-	 * @param ElggSession          $session       Session service
-	 * @param Translator           $translator    Translator
-	 * @param EntityTable          $entities      Entity table
-	 * @param LoggerInterface      $logger        Logger
+	 * @param Queue              $queue   Queue
+	 * @param PluginHooksService $hooks   Plugin hook service
+	 * @param ElggSession        $session Session service
```

Comment:
check the `uses` at the start of the file for the ones you removed from the class

Reply:
done